worker/eid-sync.js

```diff
@@ -8,7 +8,7 @@ var
 
 var Sync = {
 
-  timeout: 3000,
+  timeout: 30000,
```

Comment:
This should be taken to a config file, to avoid redeploying every time we want to change this value